### ISSUE TITLE
vget_low_s8 behind feature flag on beta

### ISSUE THREAD

**Issue by dragostis** [rust-lang/rust#94083]
While using the `aarch64` intrinsics that are stable on beta I've noticed that [`vget_low_s8`](https://doc.rust-lang.org/core/arch/aarch64/fn.vget_low_s8.html#) still complains about not being given the `stdsimd` feature.

```rust
use std::arch::aarch64::*;

fn main() {
    [
        unsafe { vmovl_s8(vget_low_s8(val.0)) },
        unsafe { vmovl_s8(vget_high_s8(val.0)) },
    ];
}
```

### Meta

`rustc --version --verbose`:
```
rustc 1.59.0-beta.8 (1945ce657 2022-02-12)
binary: rustc
commit-hash: 1945ce6579506787e0b18f0a2ea03fdb4dfc81c7
commit-date: 2022-02-12
host: x86_64-apple-darwin
release: 1.59.0-beta.8
LLVM version: 13.0.0
```

<details><summary>Backtrace</summary>
<p>

```
error[E0658]: use of unstable library feature 'stdsimd'
```

</p>
</details>

**Comment by Amanieu** [comment]
Yes, it seems this was overlooked in the stabilization. It's an easy fix, but probably not worth backporting at the last minute. In the meantime it should be possible to work around using some of the other intrinsics.

**Comment by jieyouxu** [comment]
Triage: stabilization fixed in stdarch submodule bump inside #94529.
